items/tests.py

```diff
@@ -65,9 +64,52 @@ def test_get_not_owner(self):
      authentication=self.setSession('user2', 123)))
 
   def test_put_not_owner(self):
-    self.assertHttpUnauthorized(self.api_client.put(self.url + '1/', format='json', 
-      data={}, authentication=self.setSession('user2', 123))) 
+    self.assertHttpUnauthorized(self.api_client.put(self.url + '1/', format='json',
+      data={}, authentication=self.setSession('user2', 123)))
 
   def test_delete_not_owner(self):
-    self.assertHttpUnauthorized(self.api_client.delete(self.url + '1/', format='json', 
-      authentication=self.setSession('user2', 123)))    
+    self.assertHttpUnauthorized(self.api_client.delete(self.url + '1/', format='json',
+      authentication=self.setSession('user2', 123)))
+
+
+class LoginResourceTest(ResourceTestCase):
+
+  def setUp(self):
+    super(LoginResourceTest, self).setUp()
+    self.login_url = '/api/v1/auth/login/'
+    self.logout_url = '/api/v1/auth/logout/'
+    user = User.objects.create_user('user1', 'user1@mail.ru', 123)
```

Comment:
just `User.objects.create_user('user1', 'user1@mail.ru', 123)` is normal here for me